topics/phaser/index.md

```diff
@@ -0,0 +1,11 @@
+---
+aliases: phaserjs
+created_by: Richard Davey
+display_name: Phaser
+github_url: https://github.com/photonstorm/phaser
+logo: phaser.png
+short_description: Phaser is a fun, free and fast 2D game framework for making HTML5 games for desktop and mobile web browsers, supporting Canvas and WebGL rendering.
```

Comment:
Think this is a wee bit long. Can you shorten down to ~75 chars?

Reply:
In #86, I added a test to check the length of short_description. I also bumped the length to 130 characters based on how long existing topic short_descriptions are.

Reply:
If we did want to shorten this, I think this reads well: "Phaser is a fun, free and fast 2D game framework for making HTML5 games for desktop and mobile web browsers."